Cargo.toml

```diff
@@ -8,18 +8,16 @@ edition = "2018"
 
 [dependencies]
 bit-vec = "0.6"
-parking_lot = "0.11.1"
-futures = "0.3.9"
-tokio = { version = "0.2.21", features = ["macros", "rt-threaded", "time"] }
-mini-redis = "0.4"
-derive_more = "0.99.7"
-serde = { version = "1.0.110", features = ["derive"] }
+futures = "0.3"
 futures-timer = "3.0.2"
-log = "0.4.14"
-env_logger = "0.8.3"
+derive_more = "0.99"
+log = "0.4"
+rand = "0.8"
+async-trait = "0.1"
 codec = { package = "parity-scale-codec", version = "2", default-features = false, features = ["derive"] }
-rand = "0.8.3"
-async-trait = "0"
 
 [dev-dependencies]
 libp2p = { version ="0", features = ["mdns", "tcp-tokio"]}
+tokio = { version = "0.2.21", features = ["macros", "rt-threaded", "time"] }
```

Comment:
Do we need macros and time now?

Reply:
"time" was not needed but `macros` together with `rt` and `rt-multi-thread` are needed for tests because of `#[tokio::test]`.